addon/components/pix-input-code.js

```diff
@@ -0,0 +1,69 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+const ERROR_MESSAGE = 'ERROR in PixInputCode component, you must provide an @ariaLabel';
+
+export default class PixInputCode extends Component {
+  get numIterations() {
+    const length = this.args.numInputs || 6
+    return Array.from({ length }, (_, i) => i + 1)
+  }
+
+  get ariaLabel() {
+    if (!this.args.ariaLabel) {
+      throw new Error(ERROR_MESSAGE);
+    }
+    return this.args.ariaLabel;
+  }
+
+  get inputType() {
+    return this.args.inputType || 'number';
+  }
+
+  get placeholder() {
+    return !this.args.inputType || this.args.inputType === 'number' ? '0' : null;
```

Comment:
**proposition**
Je me suis fais un noeud au cerveau, mais j'ai l'impression que `!this.args.inputType` n'est pas obligatoire non ? Si on ne fournis pas `inputType` alors on tombera sur `0` non ? 
```js
return this.args.inputType === 'number' ? '0' : null;
```

Reply:
no si inputType est undefined du coup il n'est pas = a 0 et donc le placeholder sera null